portfolio/src/main/webapp/style.css

```diff
@@ -7,3 +34,67 @@
 #greeting-container {
   margin-top: 20px;
 }
+#cat-image{
+  height: 250px;
+  margin: 30px;
+  transition: transform .5s;
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+#cat-image:hover{
+  transform: scale(1.5);
+}
+
+#quote-container{
+    font-style: italic;
+}
+
+.center {
```

Comment:
nit: consistent spacing between tag and "{" in this file.  Sometimes it is 1 space, sometimes it is none.